lib/instrumentation/undici.js

```diff
@@ -100,6 +102,15 @@ module.exports = function addUndiciChannels(agent, undici, modName, shim) {
     for (const key in outboundHeaders) {
       request.addHeader(key, outboundHeaders[key])
     }
+
+    const name = NAMES.EXTERNAL.PREFIX + UNKNOWN_HOST + (request.method || 'get')
```

Comment:
Given this style of doing things will be newer for externals, I'm thinking it would be useful to add trace level logging here for "creating external as unknown" and then later in the other event for "updating external to X". I'm somewhat curious what the existing logging shows, as well. I can imaging we may get some messages that are hard to tell why we can't add a segment because we don't have the context. Usually we'll have a name to go off of to find the external call but won't be able to here. That may just be a limitation we are stuck with based on how undici is written.

Reply:
Scanning the existing and http-outbound instrumentation, I don't think we typically include the verb in the name itself? As such, I don't think we should do so here unless I'm missing something.

If we were to include it, I'm guessing we'd want to use 'GET'.

Also, just want to double check `request.method` is available here? If it is but not specified means default to 'GET' (and below) then cool. If it always isn't available here, then defaulting to 'GET' would not actually be awesome cause we don't know. Prob is the case but just want to double-check.